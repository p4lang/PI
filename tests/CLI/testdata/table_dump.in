table_add ipv4_lpm 10.0.0.1/12 => set_nhop 10.0.0.1 13
table_dump ipv4_lpm
table_set_default ipv4_lpm set_nhop 8.8.8.8 9
table_add ipv4_lpm 192.0.0.0/8 => set_nhop 192.168.0.1 4
table_dump ipv4_lpm
table_delete ipv4_lpm 0
table_dump ipv4_lpm
table_reset_default ipv4_lpm
table_delete ipv4_lpm 1
table_dump ipv4_lpm
