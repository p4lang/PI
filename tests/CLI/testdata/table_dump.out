????
Device assigned successfully.
Selecting device.
????
Entry was successfully added with handle 0.
????
Successfully retrieved 1 entrie(s).
==========
TABLE ENTRIES
**********
Dumping entry 0
Match key:
* ipv4.dstAddr        : LPM       0a000001/12
Action entry: set_nhop - 0a000001, 0013
==========
Dumping default entry
EMPTY
==========
????
Default entry was successfully set.
????
Entry was successfully added with handle 1.
????
Successfully retrieved 2 entrie(s).
==========
TABLE ENTRIES
**********
Dumping entry 0
Match key:
* ipv4.dstAddr        : LPM       0a000001/12
Action entry: set_nhop - 0a000001, 0013
**********
Dumping entry 1
Match key:
* ipv4.dstAddr        : LPM       c0000000/8
Action entry: set_nhop - c0a80001, 0004
==========
Dumping default entry
Action entry: set_nhop - 08080808, 0009
==========
????
Entry with handle 0 was successfully removed.
????
Successfully retrieved 1 entrie(s).
==========
TABLE ENTRIES
**********
Dumping entry 1
Match key:
* ipv4.dstAddr        : LPM       c0000000/8
Action entry: set_nhop - c0a80001, 0004
==========
Dumping default entry
Action entry: set_nhop - 08080808, 0009
==========
????
Default entry was successfully reset.
????
Entry with handle 1 was successfully removed.
????
Successfully retrieved 0 entrie(s).
==========
TABLE ENTRIES
==========
Dumping default entry
EMPTY
==========
????
